app/views/staff/shared/projects/_table.html.haml

```diff
@@ -1,11 +1,19 @@
 - unless projects.empty?
-  %table.govuk-table.transactions
+  %table.govuk-table.projects
+    %caption.govuk-table__caption.govuk-visually-hidden
+      Projects
```

Comment:
Internationalise me!

Reply:
Boo! You're obsessed! I find our insitance on using locales adds a massive mental overhead for my little brain!

I will however take one for the team!